ui/react-components/constants.js

```diff
@@ -11,9 +11,17 @@ export const availableForAppointments = "Available for appointments";
 export const locationUrl = `${restWestV1}/location`;
 export const specialityUrl = `${restWestV1}/speciality/all`;
 export const appointmentsSaveUrl = `${restWestV1}/appointments`;
+export const recurringAppointmentsSaveUrl = `${restWestV1}/recurring-appointments`;
 
 export const appName = 'appointments';
 
 export const BAHMNI_CONFIG_URL = `${hostUrl}/bahmni_config/openmrs/apps`;
 export const IMPLEMENTATION_CONFIG_URL = `${hostUrl}/implementation_config/openmrs/apps`;
 export const locationTagName = 'Appointment Location';
+export const minDurationForAppointment = 30;
```

Comment:
So is it not configurable? How it behaves in older appointments?

Reply:
in old appointments also it is default 30 mins (not from configuration)